### DISCUSSION TITLE
Read spreedsheet from budibase

### DISCUSSION THREAD

**Discussion by jaarroyl** [Budibase/budibase#8409] [Help] [answered]
Hi guys!

Im trying to connect and spreadsheet using Google Sheets conector but even I gave permission and copy the URL as the guide says it runs in error according to the image.

What should I do!! best

![image](https://user-images.githubusercontent.com/41343847/198077340-cdf55e7a-9c36-4561-af32-92582b1715a8.png)

**Comment by melohagan** [comment, marked as answer]
Hey @jaarroyl 

Is your spreadsheet a Google native one, or is it in another format such as `xlsx` ?

**Reply by jaarroyl** [reply]
It is a xlsx upload to google docs!

**Reply by jaarroyl** [reply]
I create from spreadsheet directly and works! Why does it happen? Thanks
